src/lib/temple/back/dryrun.ts

```diff
@@ -32,11 +32,14 @@ export async function dryRunOpParams({ opParams, networkRpc, sourcePkh, sourcePu
         tezos.contract
           .batch(formated)
           .send()
-          .catch(() => undefined),
-        tezos.estimate.batch(formated).catch(() => undefined)
+          .catch(e => ({ ...e, isError: true })),
+        tezos.estimate.batch(formated).catch(e => ({ ...e, isError: true }))
       ]);
+      if (result.some(x => x.isError)) {
+        return result;
```

Comment:
please, refactor `dryRunOpParams` function as it's returning a lot of different types right now.
<img width="792" alt="image" src="https://user-images.githubusercontent.com/10281464/160373043-69de844c-961c-43c5-8e3b-748aa80874f9.png">

It would be perfect if it will return something like this
```
{
    result?: {} // what is returned in line 62 right now
    error?: {} // Error with explenation what is going wrong
}
```

This will remove assertions (`as any`) and ts-ignores that are present right now